build.gradle

```diff
@@ -195,6 +206,24 @@ project(':aeron-samples') {
     build.doLast {
         tasks.shadowJar.execute()
     }
+
```

Comment:
I think Phil was meaning to remove those for the PR and just hadn't gotten to them.